portfolio/src/test/java/com/google/sps/PortfolioTests.java

```diff
@@ -0,0 +1,45 @@
+package com.google.sps;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import comments.Comment;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+
+@RunWith(JUnit4.class)
+public final class PortfolioTests {
```

Comment:
Maybe you would like to add a more accurate name, since probably the Portfolio will have multiple test classes, and this one is super generic.
Something like: `CommentDatastoreTest` - which will be responsible to test the comment storage tests.

Keep in mind that for readability purposes, the name of the tests and its class should be meaningful to the reader.

Reply:
Usually unit tests are written per class so in your case:
Comment -> CommentTest
DataServlet -> DataServletTest
TranslationTest -> TranslationTest